manuscript.md

```diff
@@ -104,14 +104,14 @@ We have indeed observed this in our own projects: At the time of writing, QuantM
 
 Before we delve into the role of OSS in computational proteomics, and before losing the reader in the details of the manuscript and the importance of FAIR software, we would like to highlight one of the fundamental aspects and challenges in the development of proteomics software: the use of software licenses.
 
-Different to other fields in bioinformatics, proteomics software tools and algorithms are often using custom licenses, with complex clauses and terms that confuse the users and developers.
+Different to other fields in bioinformatics, proteomics software tools and algorithms often use custom licenses, with complex clauses and terms that confuse the users and developers. [really is this common practice? I personally know of only a single project]
```

Comment:
I can compile statistics, but I know from Bioconda that when we were dealing with proteomics tools, a lot of them were custom licenses. Instead, Sam tools, data, and other big projects in genomics use standard licences. I think @bgruening from Bioconda and Galaxy can help you figure this out. I will try to compile some stats if I find time.

Reply:
fragpipe, msfragger, DIA-NN, MaxQuant, everything that uses RawFileReader and a few more I removed from my memory. It's a huge problem in Proteomics compared to Genomics.

Reply:
See issue #8